SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -195,6 +192,7 @@ void USpatialNetDriver::OnConnectedToSpatialOS()
 void USpatialNetDriver::CreateAndInitializeCoreClasses()
 {
 	SpatialOutputDevice = MakeUnique<FSpatialOutputDevice>(Connection, TEXT("Unreal"));
+	TimerManager = MakeUnique<FTimerManager>();
```

Comment:
Why is this a unique pointer? Can it not just be an instance variable that we destruct?

Reply:
Agree with this, unless I'm missing something.